tsl/test/isolation/expected/attach_chunk_isolation.out

```diff
@@ -0,0 +1,368 @@
+Parsed test spec with 4 sessions
+
+starting permutation: s3_wait_before_lock s1b s1_attach s4_insert s3_release_lock s1c s3_show_num_chunks
+step s3_wait_before_lock: SELECT debug_waitpoint_enable('chunk_attach_before_lock');
+debug_waitpoint_enable
+----------------------
+                      
+(1 row)
+
+step s1b: BEGIN;
+step s1_attach: CALL attach_chunk('attach_test', 'chunk_to_attach', jsonb_build_object('time', jsonb_build_array('2025-06-01', '2025-06-11'))); <waiting ...>
+step s4_insert: INSERT INTO attach_test VALUES ('2025-06-03 00:15', 1, 31.5);
+step s3_release_lock: SELECT debug_waitpoint_release('chunk_attach_before_lock');
+debug_waitpoint_release
+-----------------------
+                       
+(1 row)
+
+step s1_attach: <... completed>
+ERROR:  chunk creation failed due to collision
+step s1c: COMMIT;
+step s3_show_num_chunks: SELECT count(*) FROM show_chunks('attach_test');
+count
+-----
+    1
+(1 row)
+
+
+starting permutation: s1b s1_attach s3_show_num_rows s4_insert s3_release_lock s1c s3_show_num_chunks s3_show_num_rows
+step s1b: BEGIN;
+step s1_attach: CALL attach_chunk('attach_test', 'chunk_to_attach', jsonb_build_object('time', jsonb_build_array('2025-06-01', '2025-06-11')));
+step s3_show_num_rows: SELECT count(*) FROM attach_test;
+count
+-----
+    0
+(1 row)
+
+step s4_insert: INSERT INTO attach_test VALUES ('2025-06-03 00:15', 1, 31.5); <waiting ...>
+s3: WARNING:  you don't own a lock of type ExclusiveLock
```

Comment:
Hmm, this doesn't look right..... I believe this warning means something is trying to release an ExclusiveLock that it doesn't have. You probably need to double-check this.